docs/user/reference/metrics/test.c

```diff
@@ -0,0 +1,194 @@
+
```

Comment:
❓

Reply:
Must have had some random new file I opened in VSCode and accidentally committed it.

Reply:
Oh, haha I know what it is...  it's a bitmap of a bird logo